pkg/apis/minio.rook.io/v1alpha1/types.go

```diff
@@ -55,8 +55,8 @@ type ObjectStoreSpec struct {
 	// tolerations).
 	Placement rook.PlacementSpec `json:"placement,omitempty"`
 
-	// Port of operation for the Minio object store.
-	Port int32 `json:"port"`
```

Comment:
are upgrades supported with minio? what happens for existing clusters that have set this port?

Reply:
upgrades haven't really been supported for minio so far, the types are still in alpha and there is no upgrade guide.  I'd probably take the stance myself that we should still have the freedom to change/evolve the types while they are still in alpha.  I don't think the cost of writing/testing migration logic would really be justified, what are your thoughts? Do you have a strong opinion?

Reply:
one option would be to have a default port of 9000.  If an existing minio spec doesn't have the new NetworkSpec format, then we could fall back to a default port, so their minio object store spec is still valid?